sdk/rust/Cargo.toml

```diff
@@ -1,16 +1,10 @@
 [workspace]
-members = [
-  "crates/dagger-codegen",
-  "crates/dagger-sdk",
-  "crates/dagger-bootstrap",
-]
+members = ["crates/*"]
 resolver = "2"
 
 [workspace.package]
-version = "0.3.2"
+version = "0.3.3"
```

Comment:
Should we have the release numbers follow the same versioning that the other SDKs do? So that this would be v0.9.5 now, then the engine versions and the sdk versions all match.

Not a blocker for this particular PR, since it's been like this for the last few releases, but just noticed it right now!

cc @helderco

Reply:
Yep. We could do that. I guess it is still a left over from when rust sdk was by itself

Reply:
I'll do a follow up pr with the alignment when I bump the engine the next time